shared/contracts/BaseTemplate.sol

```diff
@@ -257,4 +281,12 @@ contract BaseTemplate is APMNamehash, IsContract {
     function _ensureAragonIdIsValid(address _aragonID) internal view {
         require(isContract(address(_aragonID)), ERROR_ARAGON_ID_NOT_CONTRACT);
     }
+
+    /* HELPERS */
+
+    // TODO: Move to helper (in this repo for now)
+    function _toAddress(uint256 _value) internal pure returns(address) {
+        require(_value <= uint160(-1), ERROR_CANNOT_CAST_VALUE_TO_ADDRESS);
+        return address(_value);
```

Comment:
Does solidity overflow if the value is too big?

Reply:
yup